flow/SystemMonitor.cpp

```diff
@@ -182,6 +180,8 @@ SystemStatistics customSystemMonitor(std::string eventName, StatisticsState *sta
 				itr.maxDuration = 0;
 			}
 
+			g_network->networkInfo.metrics.lastZeroBusy = std::min(currentStats.elapsed,g_network->networkInfo.metrics.starvationTrackers[0].duration)/currentStats.elapsed;
```

Comment:
It looks like duration is being assigned 0 at the end of the above loop, so I think this is going to report 0 every time. I guess we could put this line in the loop and guard it by a check if the tracker being iterated has priority 1. Or we could do it on the first time through the loop, though we might want an `ASSERT` just to make sure we keep priority 1 as the first item in the vector.

One other nit is that the name `lastZeroBusy` may be slightly confusing based on what we're measuring. It might be clearer to name it something like `lastRunLoopBusyness`, etc.